opus/import/obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py

```diff
@@ -0,0 +1,246 @@
+################################################################################
+# obs_bundle_cassini_uvis_solarocc_beckerjarmak2023.py
+#
+# Defines the ObsBundleCassiniUvisSolarOccBeckerJarmak class, which encapsulates
+# fields in the common, common occultation, obs_mission_cassini, and obs_instrument_couvis
+# tables for COUVIS solar occultations. This class supports derived data from UVIS EUV
+# archived in cassini_uvis_solarocc_beckerjarmak2023.
+################################################################################
+
+from datetime import datetime, date
+from obs_bundle_occ_common import ObsBundleOccCommon
+from obs_cassini_common_pds4 import ObsCassiniCommonPDS4
+
+class ObsBundleCassiniUvisSolarOccBeckerJarmak(ObsBundleOccCommon, ObsCassiniCommonPDS4):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def _inst_name(self):
+        return 'Cassini UVIS'
+
+
+    #############################
+    ### OVERRIDE FROM ObsBase ###
+    #############################
+
+    @property
+    def instrument_id(self):
+        return 'COUVIS'
+
+
+    ################################
+    ### OVERRIDE FROM ObsGeneral ###
+    ################################
+
+    # Since the RA and dec of th Sun (not a point source) moves relative to the spacecraft (unlike distant stars),
+    # use None, i.e. don't override obs_profile.py's _prof_ra_dec_helper()
+
+    def field_obs_general_planet_id(self):
+        return self._create_mult('SAT')
+
+    def _target_name(self):
+        return [('S RINGS', 'Saturn Rings')]
+
+
+    ################################
+    ### OVERRIDE FROM ObsProfile ###
+    ################################
+
+    def field_obs_profile_occ_type(self):
+        return self._create_mult('SOL')
+
+    def field_obs_profile_source(self):
+        return self._create_mult('Sun')
+
+    def field_obs_profile_host(self):
+        return self._create_mult('cassini')
+
+    def field_obs_profile_temporal_sampling(self):
+        return self._index_col('rings:temporal_sampling') # sec
+
+    def field_obs_profile_wl_band(self):
+        return self._create_mult('UV')
+
+
+    #####################################
+    ### OVERRIDE FROM ObsRingGeometry ###
+    #####################################
+
+    def field_obs_ring_geometry_projected_radial_resolution1(self):
```

Comment:
Do we need these since they default to None?

I think we actually could calculate these from the tab files by looking for the min and max delta between ring radius from one sample to the next, but probably not worth the effort. Perhaps it would be worth adding it to the wish list for the index tools.